data/blog/2024-year-in-review.mdx

```diff
@@ -0,0 +1,545 @@
+companies, or direct crowd funding, or the state?
+
+From the beginning, the vision of OpenSats was to create a sustainable
+foundation that provides [consistent funding](/about) for free and open-source
+projects, especially those who make bitcoin flourish. We don't want to be
+dependent on quarterly profits nor on funding from the state, which is why we
+chose a public charity as a vehicle.
+
+As mentioned in our [mission statement](/mission), we believe that we can help
+make bitcoin more accessible, robust, secure, and decentralized. Further, we
+believe that the projects we support have the potential to revolutionize the way
+we think about individual sovereignty and liberty—values that we hold in high
+regard. We want to operate with as little overhead as possible, which is why we
+don't take a cut from donations to our funds. And we don't want any one person
+or small group to have control over funding or organizational decisions, which
+is why we have a 9-person board that is the final arbiter of all decisions.
+
+We don't claim to have all the answers, and we don't claim that what we are
+doing is frictionless or perfect. We are bound to make mistakes, and we will
+always be prone to false positives (funding projects that shouldn't receive
+funding) as well as false negatives (not funding projects that should receive
+funding). We strive to gradually and continually improve our processes, and we
+hope to get better at what we are doing with every passing year. That said, we
+are incredibly pleased to see that our structure and processes are working, and
+we believe this year in review is a testament to that.
+
+---
+
+## Announcements
+
+At the beginning of the year, we announced long-term support for [René
+Pickhardt](/blog/rene-pickhardt-receives-lts-grant) and
+[0xB10C](/blog/0xB10C-receives-lts-grant), two researchers/developers with a
+major track record, both doing important work related to bitcoin infrastructure.
+Soon after we announced long-term support for [Bruno
+Garcia](/blog/bruno-garcia-receives-lts-grant), another developer that qualifies
+as a [load-bearing internet
+person](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors)
+by our estimation. In May [Shashwat
+Vangani](/blog/shashwat-vangani-receives-lts-grant) joined our roster of LTS
+grantees, followed by [Calle](/blog/cashu-calle-receives-lts-grant) in June, and
+[Jason Donenfeld](/blog/jason-donenfeld-lts-grant)
+&
+[Jon Atack](/blog/jon-atack-receives-lts-grant)
+in September. The last two long-term
+support grants in 2024 were awarded to [Dusty
+Daemon](/blog/dusty-daemon-receives-lts-grant) and
+[m1sterc001guy](/blog/m1sterc001guy-receives-lts-grant), as announced in October
+and November, respectively. This brings the total number of grantees receiving
+long-term support from our General Fund to 18, adding to the 9 announced in 2023
+([Marco](/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors),
```

Comment:
Just flagging this line in case the link does actually exist? Could not find it in the codebase though.

Reply:
That's the correct link, we announced his LTS with the [announcement of the program](https://opensats.org/blog/announcing-lts-grant-program-to-support-bitcoin-core-contributors):

> The first recipient of an OpenSats LTS Grant is Marco Falke, a long-term maintainer and contributor of Bitcoin Core [...]